### DISCUSSION TITLE
Wrong email-requirements for ISM Germany

### DISCUSSION THREAD

**Discussion by acsig** [community/community#51836] [GitHub Education]
Hello, 

I failed to register as a teacher, because for the German International School of Management (ISM) the email-requirements are

- *student.ism.de 
- *ism-dortmund.de

Teachers emails are *.dozent.ism.de

Could you add this subdomain as well? 

Thanks: Arne

**Comment by samus-aran** [comment]
Hi @acsig 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍
